test/mocks/mock_shared_preferences.dart

```diff
@@ -0,0 +1,102 @@
+import 'package:rx_shared_preferences/rx_shared_preferences.dart';
```

Comment:
wasted a lot of time trying to make shared preferences work - I think shared preferences simply don't work in unit tests, so they can be replaced with this.